docs/topics/migrations.txt

```diff
@@ -218,6 +218,23 @@ Apps without migrations must not have relations (``ForeignKey``,
 ``ManyToManyField``, etc.) to apps with migrations. Sometimes it may work, but
 it's not supported.
 
+Swappable dependencies
+----------------------
+
+.. function:: django.db.migrations.swappable_dependency(value)
+
+The ``swappable_dependency()`` function is used by migration operations to
+declare "swappable" dependencies such as swappable user models. The argument
+``value`` is a string ``"<app label>.<model>"`` describing an app label and a
+model name, e.g. ``"myapp.MyModel"``.
+
+By using ``swappable_dependency()``, you inform the migration framework that
+the migration operation relies on a swappable model, allowing for the
+possibility of substituting the model with a different implementation in the
+future. This is typically used for referencing models that are subject to
+customization or replacement, such as the custom user model
+(``settings.AUTH_USER_MODEL``) in Django's authentication system.
```

Comment:
I think the distinction needs to be made -- I think `settings.AUTH_USER_MODEL` looks close enough to the `<app>.<Model>` pattern, that if we don't hammer this point in, some people will be confused.

```suggestion
(``settings.AUTH_USER_MODEL``, which defaults to ``"auth.User"``) in Django's authentication system.
```